test/core/table_init.wast

```diff
@@ -33,9 +36,13 @@
   (func (result i32) (i32.const 8))
   (func (result i32) (i32.const 9))  ;; index 9
   (func (export "test")
-    (table.init $t0 1 (i32.const 7) (i32.const 0) (i32.const 4)))
+    (table.init $t0 2 (i32.const 7) (i32.const 0) (i32.const 4)))
+  (func (export "test-t64")
+    (table.init $t64 2 (i32.const 7) (i32.const 0) (i32.const 4)))
```

Comment:
I think this trailing `i32.const` needs to be `i64.const`?

Reply:
Thanks for catching.. taking a look now.